portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,6 +29,41 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    ArrayList<String> messages = new ArrayList<String>();
+    messages.add("How's it going?");
+    messages.add("My name is Emma.");
+    messages.add("I'm a junior at Rice.");
+    String json = convertToJson(messages);
+
+    // Send the JSON as the response
+    response.setContentType("application/json;");
+    response.getWriter().println("Hello world!");
+    response.getWriter().println(json);
+  }
+
+  /**
+   * Converts a ServerStats instance into a JSON string using manual String concatentation.
+   */
+  private String convertToJson(ArrayList<String> messages) {
```

Comment:
This produces the following output:

```{How's it going? My name is Emma. I'm a junior at Rice.}```

It's not quite JSON. Is this intended?